tasks/system/Linux.yml

```diff
@@ -1,117 +1,126 @@
-    become: False
-    register: sonar_artifact
 
-  - name: "Install SonarQube {{ sonar_major_version }}.{{ sonar_minor_version }}"
+  - name: 'Install SonarQube {{ sonar_major_version }}.{{ sonar_minor_version }}'
     unarchive:
-      src: "{{ sonar_artifact.dest }}"
+      src: '{{ download_path }}sonarqube-{{ sonar_major_version }}.{{ sonar_minor_version }}.zip'
       exclude: "{{ sonar_exclude_plugins }} + \
-      [ 'sonarqube-{{ sonar_major_version }}.{{ sonar_minor_version }}/conf/sonar.properties' ]"
-      dest: "{{ sonar_path }}"
-      list_files: True
-    become: True
-    become_user: "{{ sonar_user }}"
-    register: sonar_installation_result
-    # workaround for ansible 2.6.*, see https://github.com/ansible/ansible/issues/45124
-    ignore_errors: True
-
-  - name: "SonarQube installation facts"
-    set_fact:
-      sonar_installation: "{{ sonar_path }}/{{ sonar_installation_result.files.0 }}"
-      distro: "{{ ansible_facts.system | lower }}"
-      arch: "{{ 'x86-32' if ansible_architecture == 'i386' else 'x86-64' }}"
-    when: sonar_installation_result.files is defined
-
-    # continuation of ansible 2.6.* workaround
-  - name: "SonarQube installation facts"
-    set_fact:
-      sonar_installation: "{{ sonar_path }}/\
-      sonarqube-{{ sonar_major_version }}.{{ sonar_minor_version }}/"
-      distro: "{{ ansible_facts.system | lower }}"
-      arch: "{{ 'x86-32' if ansible_architecture == 'i386' else 'x86-64' }}"
-    when: sonar_installation_result.files is not defined
-
-  - name: "Render sonar.properties"
+      [ 'sonarqube-{{ sonar_major_version }}.{{ sonar_minor_version }}/conf/sonar.properties', \
+      'sonarqube-{{ sonar_major_version }}.{{ sonar_minor_version }}/temp/README.txt']"
+      dest: '{{ sonar_path }}'
+      owner: '{{ sonar_user }}'
+      group: '{{ sonar_group }}'
+      remote_src: True
+      creates: '{{ sonar_installation }}conf/wrapper.conf'
+    notify: Restart Sonarqube
+
+  - name: Render sonar.properties
     template:
-      src: "sonar.properties.j2"
-      dest: "{{ sonar_installation }}conf/sonar.properties"
-      mode: "0400"
-      owner: "{{ sonar_user }}"
+      src: sonar.properties.j2
+      dest: '{{ sonar_installation }}conf/sonar.properties'
+      mode: 0400
+      owner: '{{ sonar_user }}'
+    notify: Restart Sonarqube
 
-  - name: "Render sonarqube.service"
+  - name: Render sonarqube.service
     template:
-      src: "sonarqube.service.j2"
-      dest: "/etc/systemd/system/sonarqube.service"
-      mode: "0444"
-      owner: "{{ sonar_user }}"
-
-  - name: "Systemd daemon-reload"
-    systemd:
-      daemon_reload: True
-
-  - name: "Enable sonarqube.service"
-    service:
-      name: "sonarqube"
-      enabled: True
+      src: sonarqube.service.j2
+      dest: /etc/systemd/system/sonarqube.service
+      mode: 0444
+      owner: '{{ sonar_user }}'
+    notify: Restart Sonarqube
 
-  - name: "Check nginx conf directory"
+  - name: Check nginx conf directory
```

Comment:
Why do we need explicit check?

Reply:
Removed because of use {{ nginx_config_path }} variable